elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -337,6 +337,14 @@ public String getRejectionLogMessage(Map<String, ModelNode> attributes) {
                 .setDiscard(DiscardAttributeChecker.ALWAYS, ElytronDescriptionConstants.ONLY_LEAF_CERT)
                 .setDiscard(DiscardAttributeChecker.ALWAYS, ElytronDescriptionConstants.SOFT_FAIL)
                 .end();
+        builder.addChildResource(PathElement.pathElement(ElytronDescriptionConstants.KEY_STORE))
+                .addOperationTransformationOverride(ElytronDescriptionConstants.READ_ALIAS)
+                .setDiscard(DiscardAttributeChecker.ALWAYS, ModifiableKeyStoreDecorator.ReadAliasHandler.VERBOSE).end();
+        builder.addChildResource(PathElement.pathElement(ElytronDescriptionConstants.KEY_STORE))
+                .addOperationTransformationOverride(ElytronDescriptionConstants.READ_ALIASES)
+                .setDiscard(DiscardAttributeChecker.ALWAYS, ModifiableKeyStoreDecorator.ReadAliasesHandler.VERBOSE)
+                .setDiscard(DiscardAttributeChecker.ALWAYS, ModifiableKeyStoreDecorator.ReadAliasesHandler.RECURSIVE).end();
```

Comment:
You can also add:

`.addRejectCheck(new RejectAttributeChecker.SimpleRejectAttributeChecker(ModelNode.TRUE), ElytronDescriptionConstants.VERBOSE)`
`.addRejectCheck(new RejectAttributeChecker.SimpleRejectAttributeChecker(ModelNode.TRUE), ElytronDescriptionConstants.RECURSIVE)`